lib/Field/styles.scss

```diff
@@ -47,12 +47,32 @@ $field-data-color: $neutral-base !default;
 .field__instructions {
   font-style: italic;
   margin: $field-spacing-base 0 0;
+  padding: $field-spacing-base 0 0 0;
+  border-top: 1px solid $neutral-light;
+  transition: all 0.2s ease; // Does this go here?
 }
 
 .field__content {
-  padding-left: $field-spacing-base;
+  background: #F7F9FA;
+  border:1px solid transparent;
+  border-radius: 4px;
+  padding: 30px; // Should be a variable?
```

Comment:
I would say yes. Even though it seems strange at first if we change the global spacing in the future this should effect all components.

To get 30px you could do `$layout-spacing-base*1.5`.

Reply:
Awesome, will do that now!

Reply:
Another idea is that we could add some new settings for spacing 😄 

e.g.

```
$layout-spacing-base: 20px;
$layout-spacing-small: $layout-spacing-base/2;
$layout-spacing-large: $layout-spacing-base*1.5;
$layout-spacing-larger: $layout-spacing-base*2;
```